config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -261,10 +261,10 @@ periodics:
     preset-k8s-ssh: "true"
   spec:
     containers:
-    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230207-192d5afee3-master
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230207-192d5afee3-1.25
       args:
       - --root=/go/src
-      - --repo=k8s.io/kubernetes=release-1.22
+      - --repo=k8s.io/kubernetes=release-1.25
       - --repo=github.com/containerd/containerd=release/1.5
```

Comment:
@SergeyKanzhelev @mikebrow since containerd 1.5 will be out of support on 28th feb, do we still wan to keep 1.5 job?
https://containerd.io/releases/

I was thinking runing k8s 1.25 job on containerd 1.6, master on 1.6 later add one more job with containerd 1.7+k/k master when container 1.7 rc gets cut?

Reply:
snds good when it's cut.. we can merge this till then